test/integration-new/cni/cni_version_test.go

```diff
@@ -0,0 +1,95 @@
+package cni
+
+import (
+	"fmt"
+	"time"
+
+	k8sUtils "github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/utils"
+	"github.com/aws/aws-sdk-go/service/eks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	describeAddonOutput *eks.DescribeAddonOutput
+	err                 error
+	initialCNIVersion   string
+	finalCNIVersion     string
+)
+
+var _ = Describe("test cluster upgrade/downgrade", func() {
+
+	It("should apply initial addon version successfully", func() {
+		By("getting initial cni version")
+		initialCNIVersion = f.Options.InitialCNIVersion
+		ApplyAddOn(initialCNIVersion)
```

Comment:
We should validate initialCNIVersion and finalCNIVersion that they are not empty.